### ISSUE TITLE
GitManual: Incorrect 'More at' URL

### ISSUE THREAD

**Issue by rasikapohankar** [duckduckgo/zeroclickinfo-fathead#759]
The 'More at' link shows `git-scm` but its pointing to `http://csurs.csr.uky.edu/cgi-bin/man/man2html?1+query` which gives a 404.

Example, the More at link for https://duckduckgo.com/?q=git+cherry+pick&t=canonical&ia=about points to
http://csurs.csr.uky.edu/cgi-bin/man/man2html?1+git-cherry-pick.

------
IA Page:  http://duck.co/ia/view/git_manual 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @samskeller

**Comment by pjhampton** [comment]
It's pointing to the correct place for me. What is your location setting? 

![screen shot 2017-02-08 at 8 00 13 pm](https://cloud.githubusercontent.com/assets/8960296/22754986/3f3b4346-ee39-11e6-97a9-7e4a7c30a0d0.png)

**Comment by rasikapohankar** [comment]
@pjhampton The location was set to India. I restarted my browser and now it's pointing to the correct URL for me too.

Its looking like this now: 
![screenshot](https://cloud.githubusercontent.com/assets/8025692/22755565/b74418ec-ee69-11e6-9834-81adc52b6241.png)

Earlier, the result was showing a single line with a scroll bar. Unfortunately, I did not capture a screen shot. Apologies.

I'll close this.
